### PR TITLE
GT Tool Fixes and ore balance config

### PR DESCRIPTION
**Pull request by Minepolz320**
Fixes that the axe and saw do not break ladders
- ability to toggle ore multipliers for ore Nether,End,Dense Ores
- sneaking with a scythe/plow allows you to break only 1 block
- Revert https://github.com/GTNewHorizons/GT5-Unofficial/pull/2453/commits/64c9764bd2df29be60bd27ab447296518fd0e75b
because of https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8480

### CODE REVIEW

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingOre.java [hunk 1]
```diff
@@ -35,11 +36,22 @@ public ProcessingOre() {
...
+        // Dense ore
+        if (GT_Mod.gregtechproxy.mRichOreYieldMultiplier) {
+            tIsRich = (aPrefix == OrePrefixes.oreRich) || (aPrefix == OrePrefixes.oreDense);
+        }
+        // NetherOre
+        if (GT_Mod.gregtechproxy.mNetherOreYieldMultiplier && !tIsRich) {
+            tIsRich = (aPrefix == OrePrefixes.oreNetherrack) || (aPrefix == OrePrefixes.oreNether);
+        }
+        // EndOre
+        if (GT_Mod.gregtechproxy.mNetherOreYieldMultiplier && !tIsRich) {
```
Comment (chochem) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingOre.java:52]: copy paste typo

Comment (Minepolz320) [reply src/main/java/gregtech/loaders/oreprocessing/ProcessingOre.java]: Thank you!

**Comment by Minepolz320** [comment]
Damn, someone broke the logic of the tools' behavior

**Comment by Minepolz320** [comment]
again!

**Comment by Minepolz320** [comment]
Reverted changes that broke the logic of the tools

I think we need to come up with something about this
and decide what is right

**Comment by Caedis** [comment]
Instead of saying "broke the logic", do you want to explain what you mean?

**Comment by Minepolz320** [comment]
@Caedis All the logic of the tools is tied to the digging speed, which determines which block can be broken by which tool

**Comment by Minepolz320** [comment]
for example, the bug with the scythe returned

**Comment by Minepolz320** [comment]
We need to discuss how we will rework the logic of GT tools without rewriting a large amount of code to allow such changes

**Comment by Caedis** [comment]
and that bug with the scythe is?

**Comment by Minepolz320** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8480

**Comment by Caedis** [comment]
Fair enough. I was just super annoyed that no gt tools can break freaking grass or flowers.

**Comment by Minepolz320** [comment]
I'm still thinking about how to make a fix for things that break instantly, for example torches and so on

**Comment by Caedis** [comment]
1 option is a block whitelist but that could get annoying to maintain  
Another is to do something similar to what the old code did and just check if the harvestLevel is 0, but would require removing the dependent on the output of getDiggingSpeed

**Review by OneEyeMaker — APPROVED** [review]

**Pull request merged**